src/multiselect_contextmenu.js

```diff
@@ -781,8 +781,15 @@ const updateToMultiCopyToBackpack =
               workableBlocksLength++;
             }
           }
-          return `(${workableBlocksLength}) ` +
-             `${Blockly.Msg['COPY_TO_BACKPACK']} (${backpackCount})`;
+          if (workableBlocksLength > 1) {
+            if (Blockly.Msg['COPY_X_TO_BACKPACK']) {
+              return Blockly.Msg['COPY_X_TO_BACKPACK'].replace('%1', workableBlocksLength.toString());
+            } else {
+              return `${Blockly.Msg['COPY_TO_BACKPACK']} (${workableBlocksLength})`;
+            }
+          } else {
+            return Blockly.Msg['COPY_TO_BACKPACK'];
+          }
```

Comment:
Also we need to append `workableBlocksLength` to keep the same as

https://github.com/google/blockly-samples/blob/c32266b2b7685c0d4d6646e5bcedb085c435e0b2/plugins/workspace-backpack/src/backpack_helpers.ts#L115

```suggestion
          if (workableBlocksLength > 1) {
            if (Blockly.Msg['COPY_X_TO_BACKPACK']) {
              return Blockly.Msg['COPY_X_TO_BACKPACK'].replace(
                  '%1', workableBlocksLength.toString()) +
                  ` (${backpackCount})`;
            } else {
              return `(${workableBlocksLength}) ` +
                `${Blockly.Msg['COPY_TO_BACKPACK']} (${backpackCount})`;
            }
          } else {
            return Blockly.Msg['COPY_TO_BACKPACK'] + ` (${backpackCount})`;
          }
```